### ISSUE TITLE
Vaccine coverage sample size calculation

### ISSUE THREAD

**Issue by dirkschumacher** [R4EPI/surveys#3]
I see one study using the *OpenEpiMenu software*.

- [ ] Checkout the method of the software and see if/how it can be replicated in R

**Comment by epiamsterdam** [comment]
Yes. Open Epi is useful as it can be done easily and online. However only useful for samplesize calculations for proportions of coverage. Doesnt have the ability to do incidence rate (mortality rate) samplesize calculation.
